src/components/right-sidebar/RightSideBar.module.css

```diff
@@ -0,0 +1,59 @@
+div.inputField {
+  background-color: var(--COLOR-SEARCH-BACKGROUND);
+  border-radius: 5rem;
+  box-shadow: inset 0px 1px 4px var(--COLOR-SEARCH-INNER-SHADOW);
+  color: var(--COLOR-TEXT);
+  display: flex;
+  font-size: 1.7rem;
+  margin: auto;
+  padding: 0.5rem 1.5rem;
```

Comment:
Better to use px for values less than 1 rem, so that style remains constant while you change screen size

Reply:
Actually, I tested it by zooming in and out on the page and it was not making much changes ( I guess nothing noticeable), that's why to keep consistency I had decided to use rem everywhere.

It would be good if you can attach any screenshots, where you see it is making changes in the style.